### PR TITLE
fix(app): Update code for libre claim rewards

### PR DESCRIPTION
**Pull request by fagomezra**
fix(app): Update code for libre claim rewards

### CODE REVIEW

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Add the list of supported networks in the README file.

In summary: The script supports a safe mechanism to `claimrewards` for the LIBRE network and also it uses a default function for the rest of the networks

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Some things you need before getting started:
 1.  Clone this repo using `git clone --depth=1 https://github.com/eoscostarica/claim-bp-rewards.git <YOUR_PROJECT_NAME>`
 2.  Move to the appropriate directory: `cd <YOUR_PROJECT_NAME>`.
 3.  Copy and rename the `.env.example` to `.env` then update the environment variables according to your needs
+4.  For the CLAIM_NETWORK, the supports networks are: `libre`,`libre-testnet`, `antelope`, `jungle`, `telos`, `telos-testnet`, `proton`, `proton-testnet`, `wax`, `.	wax-testnet`
```
Comment (leisterfrancisco) [inline README.md]: is it correct?

`. wax-testnet`

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Comment by fagomezra** [comment]
supported networks already added to the README file

File: README.md (hunk 1 above)
Comment (fagomezra) [reply README.md]: no, i already correct them...

**Review by leisterfrancisco — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**